contracts/src/EspTokenV2.sol

```diff
@@ -44,6 +47,13 @@ contract EspTokenV2 is EspToken {
         _mint(to, amount);
     }
 
+    /// @notice Cannot renounce ownership
+    /// @dev Override renounceOwnership() to revert, preventing accidental or malicious ownership
+    /// renunciation
+    function renounceOwnership() public virtual override onlyOwner {
```

Comment:
@alysiahuggins should this go into the V1 (or also go into V1)? If we first deploy the V1 on mainnet because reward claims aren't ready and V2 depends on reward claims to be deployed it would probably be better to have this in V1.